lib/dns/resource.js

```diff
@@ -880,6 +880,7 @@ function typeToClass(type) {
 function stringToClass(type) {
   assert(typeof type === 'string');
 
+  // eslint-disable-next-line no-prototype-builtins
```

Comment:
So I just noticed this locally as well - `no-prototype-builtins` was added to ESlint somewhere along the way. But the CircleCI script uses https://github.com/bcoin-org/bslint which wraps `eslint@5.10.0`.

We could add the exception in .eslintrc.json ? Or maybe require bslint as a  devDependency? Like bmocha?


How would you feel about that? Then the `npm run lint` command could just call bslint the same way the test commands call bmocha.

Reply:
Yeah `devDependency` route sounds good.

Reply:
OK, using `bslint` now. DIdn't want to clutter deps, so latest is being installed on the CI.